spec.html

```diff
@@ -949,6 +949,7 @@
           A code unit that is in the range 0xD800 to 0xDFFF, but is not part of a surrogate pair, is interpreted as a code point with the same value.
         </li>
       </ul>
+      <p>In this specification, the phrase "the <dfn>string-concatenation</dfn> of _A_, _B_, ..." (where each argument is a String value, a code unit, or a sequence of code units) denotes the String value whose sequence of code units is the concatenation of the code units (in order) of each of the arguments (in order).</p>
```

Comment:
Actually, I'm hoping that the "sequence of code units" alternative will evaporate if I submit a PR to resolve #828.